dev/com.ibm.ws.logging_2_fat/publish/servers/com.ibm.ws.logging.messagetracerollover/bootstrap.properties

```diff
@@ -0,0 +1,14 @@
+###############################################################################
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.trace.file.name=stdout
```

Comment:
As per the alternative approach, we shouldn't set the trace file to stdout here.